src/lib.rs

```diff
@@ -236,8 +233,6 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>> SystemContext<TYPES::Consens
             transactions: Arc::default(),
             saved_leaves,
             saved_blocks,
-            // TODO this is incorrect
-            // https://github.com/EspressoSystems/HotShot/issues/560
```

Comment:
Probably should keep this comment in

Reply:
Oops, I thought this was resolved since the issue is closed. Should I reopen it?